fastify-url-data.d.ts

```diff
@@ -0,0 +1,24 @@
+
+import fastify = require('fastify');
+import { UrlObject } from 'url';
+
+interface FastifyUrlData {
+  path: string
+  host: string
+  port: number
+  query: string
+}
+
+// Extend FastifyReply with the "fastify-url-data" function
+declare module 'fastify' {
+  interface FastifyRequest {
+    urlData (): UrlObject
```

Comment:
This should use `urijs` types. I noticed after pushing, can fix post discussion in Issues 😄